athena-elasticsearch/src/main/java/com/amazonaws/athena/connectors/elasticsearch/ElasticsearchRecordHandler.java

```diff
@@ -136,8 +140,18 @@ protected void readWithConstraint(BlockSpiller spiller, ReadRecordsRequest recor
         logger.info("readWithConstraint - enter - Domain: {}, Index: {}, Mapping: {}",
                 recordsRequest.getTableName().getSchemaName(), recordsRequest.getTableName().getTableName(),
                 recordsRequest.getSchema());
+        String domain;
+        QueryBuilder query;
+        if (recordsRequest.getConstraints().isQueryPassThrough()) {
```

Comment:
where are you setting the index for the query?

The signature calls for three parameters, two only set.

Reply:
Incorporated the required changes.